src/atoms/ArrowBullet.js

```diff
@@ -0,0 +1,31 @@
+import styled, { injectGlobal } from 'styled-components';
+
+injectGlobal`
+    @font-face {
+    font-family: 'Helveticons';
+    src: url('https://styleguide.dagbladet.no/fonts/helveticons.woff');
+  }
+`;
+
```

Comment:
This is considered bad practice. Are you sure this is what you want to do?

From styled-components docs: 
> We do not encourage the use of this. Try to use it once per app at most, if you must, contained in a single file. This is an escape hatch.

Reply:
I think this is fine for font-face declarations. We might want to postpone the injection until the component is used, though - something like 
```js
const A = styled.a`[...]`;

const ArrowBullet = props => {
    injectGlobal`[...]`;

    return <A {...props>;
}
```

Another approach could be to insert a `<FontIcon>` here instead of hardcoding it - the upside to that would be that we could more easily add support for different icon fonts (a single site could choose which icon font to use).